### ISSUE TITLE
LoadLibrary dynamically and API declares

### ISSUE THREAD

**Issue by Kr00l** [WaynePhillipsEA/twinbasic#372]
**Describe the bug**
Sometimes a flat API DLL is loaded dynamically via LoadLibrary on startup and will be freed via FreeLibrary on app close.
Below example works in VB6 but not in tB.

**To Reproduce**
```
Private Declare Function LoadLibrary Lib "kernel32" Alias "LoadLibraryW" (ByVal lpLibFileName As Long) As Long
Private Declare Function FreeLibrary Lib "kernel32" (ByVal hLibModule As Long) As Long

Private Declare Sub FPDF_InitLibrary Lib "pdfium.dll" Alias "_FPDF_InitLibrary@0" ()

Public Sub Main()

    Dim hLib As Long
    hLib = LoadLibrary(StrPtr("C:\MyPath\pdfium.dll"))
        
    If hLib <> 0 Then
        FPDF_InitLibrary
        
        MsgBox "works"
        
    End If
    
    If hLib <> 0 Then
        FreeLibrary hLib
        hLib = 0
    End If
End Sub
```

**Expected behavior**
in tB the entry point of the API declare cannot be found.

**Screenshots**
![image](https://user-images.githubusercontent.com/33756666/130244984-b11438f4-6dce-43ec-93a8-8e740286695c.png)

**Desktop (please complete the following information):**
 - OS: [e.g. Windows 10]
 - twinBASIC compiler version [e.g. 0.10.4628]

**Additional context**
If needed at all I can send over the 32 bit pdfium.dll over by e-mail. (4 MB)
But it should be testable with any other flat API DLL as well.

**Comment by WaynePhillipsEA** [comment]
The problem here is essentially the same issue as #284.  The compiler is attempting to early-bind the DLL reference (before your Main code runs, or via the OS loader / DLL import table if building an EXE).  Once #284 is fixed, this should also be resolved.  As it's slightly different in context, let's leave this issue open as well until fixed.

**Comment by Kr00l** [comment]
I don't want to extra push here. Just asking when it's planned. Thx :-)

**Comment by WaynePhillipsEA** [comment]
No worries, it's coming soon.  Next up is VBP support, then it's this.

**Comment by WaynePhillipsEA** [comment]
This should now be fixed in v0.13.40.  Thanks!

**Comment by Kr00l** [comment]
Confirm. Thanks !!
